lib/configuration/variables/sources/env.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       errorCode: 'INVALID_ENV_SOURCE_ADDRESS',
     });
 
-    if (!process.env[address]) missingEnvVariables.add(address);
+    if (typeof process.env[address] === 'undefined') missingEnvVariables.add(address);
     return { value: process.env[address] || null, isPending: !isSourceFulfilled };
```

Comment:
Let's use `process.env[address] == null` check instead. Also we need to update it below so `''` is not overridden with null

Reply:
> Let's use `process.env[address] == null` check instead. Also we need to update it below so `''` is not overridden with null

Suggestions applied, is that correct?